db/versions/0023-migration.sql

```diff
@@ -0,0 +1,25 @@
+begin
+  -- lock this table before reading from it, to prevent loss of concurrent
+  -- updates when the table is dropped.  Note that this may lead to concurrent
+  -- updates failing; the important thing is that they not succeed without
+  -- taking effect.  Failed updates will be retried.
+  lock table github_access_token_table_entities;
+
+  create table github_access_tokens
+  as
+    select
+      (value ->> 'userId')::text as user_id,
+      (value ->> 'accessToken')::text as access_token,
```

Comment:
This is encrypted, so you'll need to wrap this in a crypto container on upgrade.  I think we'd also considered just dropping this table on upgrade?  The column type for an encrypted column is JSONB.

Reply:
Good spot. I've removed the data migration now.